### ISSUE TITLE
Support for 2FA

### ISSUE THREAD

**Issue by jfversluis** [tomreinartz90/simplicate-app#3]
I have enabled two-factor authentication on Simplicate, is this, or can this be supported? 🙂

**Comment by tomreinartz90** [comment]
I don't think this will be possible as the Simplicate api has no support for this.

**Comment by jfversluis** [comment]
Is possible and the Simplicate API supports it 🙂

**Comment by tomreinartz90** [comment]
@jfversluis Thanks for doing this research. I have contacted simplicate on how to implement this the correct way.
